### ISSUE TITLE
Allow resizing of canvas area

### ISSUE THREAD

**Issue by srcnuzn** [OWASP/threat-dragon#1064]
**Describe what problem your feature request solves**:

I cannot increase the canvas size. I can only increase the app's window size.

**Environment**
Version 2.2
Windows 11
Desktop App
4K Monitor

**Describe the solution you'd like**:

I want to be able to increase the canvas size to have a larger area for modeling.

**Additional context**:

![2024-08-22_11h38_57](https://github.com/user-attachments/assets/2b5c83a4-47c1-46b6-b046-debbd2106330)

**Comment by jgadsden** [comment]
many thanks for raising this enhancement @srcnuzn , it would certainly be good to have this working better than it does
planning this for version 2.4, which should be released end of this year
